packages/plugin-commands-installation/src/recursive.ts

```diff
@@ -201,6 +199,12 @@ export default async function recursive (
       const modulesDir = localConfig.modulesDir ?? opts.modulesDir
       const { manifest, writeProjectManifest } = manifestsByPath[rootDir]
       let currentInput = [...params]
+      const ignoredPackages = (manifest.pnpm?.updateConfig?.ignoreDependencies ?? [])
+      if (opts.update && params.length === 0) {
+        currentInput = [...ignoredPackages.map(pkg => `!${pkg}`), ...currentInput]
```

Comment:
why do append currentInput if it will be empty always?

Reply:
same as https://github.com/pnpm/pnpm/pull/5408#discussion_r985068886